cadence/templates/server-configmap.yaml

```diff
@@ -8,83 +8,113 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
 data:
+{{- if semverCompare "<= 0.5.5" .Values.server.image.tag }}
```

Comment:
With version there is slight change in config naming, this will accept `semver` tags but not sure how to make it resilient for non version tags eg. `master` ? :thinking:

Reply:
I think we can release a new minor version of the chart with a minimum requirement of 0.5.7 or compatible version, so this check won't be necessary after all.